src/meta/buildTags.tsx

```diff
@@ -653,7 +652,12 @@ const buildTags = (config: BuildTagsParams) => {
   if (config.additionalMetaTags && config.additionalMetaTags.length > 0) {
     config.additionalMetaTags.forEach(tag => {
       tagsToRender.push(
-        <meta key={tag.name || tag.property || tag.httpEquiv} {...tag} />,
+        <meta
+          key={`meta:${
```

Comment:
I don't know if we put a separator after the name for easier and cleaner debugging.